README.md

```diff
@@ -58,7 +58,7 @@ Some of these are actually completed(!) already, but the following items are the
 - 🧰 Easily consumable and documented REST API, so community tools and scripts can interact with Stump
 - 🌈 And more!
 
-I am very open to suggestions and ideas, so feel free to reach out if you have anything you'd like to see!
```

Comment:
This edit is not required. I *am* very open to suggestions, and I wrote it explicitly this way to convey it as such.